webapp/packages/plugin-sql-editor/src/SqlResultTabs/SqlQueryResultService.ts

```diff
@@ -163,6 +163,7 @@ export class SqlQueryResultService {
 
         // TODO: we need to dispose table model, but don't close execution context, so now we only
         const model = this.tableViewerStorageService.get(group.modelId);
+        model?.source.cancelLoadTotalCount();
```

Comment:
since there is not dispose, we need to cancel it in result set tab manually on close

Reply:
you can move cancelLoadTotalCount call to cancel fn